RELEASES.md

```diff
@@ -4,68 +4,72 @@
 
 Due to the period of time between this release and v0.0.26, many relevant
 changes and fixes may not appear in the following list.  For a complete list of
-changes, please check: https://github.com/google/kythe/commits/
+changes, please check the commit logs:
+https://github.com/google/kythe/search?q=committer-date%3A2016-11-01..2018-05-31&type=Commits
 
-Notable changes:
- - remove GRPC server and client code
+### Notable additions
```

Comment:
Please don't change the format yet.  It doesn't match at all with the rest of the log.  We're just pushing out a release to catch-up.

Reply:
Reverted the header tags.